cef/DemoWindow.rbfrm

```diff
@@ -3099,7 +3099,7 @@ End
 	#tag Event
 		Sub Action()
 		  testField.IndentVisually = me.Value
-		  testField.ClearDirtyLines
+		  'testField.ClearDirtyLines
```

Comment:
Since I wasn't sure if the whole idea was something you wanted, I left it in commented just in case you wanted to reverse that part easily.